modules/installation-configuration-parameters.adoc

```diff
@@ -1727,6 +1727,50 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the type of disk.
 |`premium_LRS` or `standardSSD_LRS`. The default is `premium_LRS`.
 
+|`platform.azure.defaultMachinePlatform.encryptionAtHost`
+|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption.
+|`true` or `false`. The default is `false`.
+
+|`platform.azure.defaultMachinePlatform.osDisk`
+|
+|
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet`
+|
+|
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.name`
+|The name of the disk encryption set that contains the encryption key from the installation prerequisites.
+|String, for example `production_disk_encryption_set`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.resourceGroup`
+|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites. This resource group should be different from the resource group where you install the cluster to avoid deleting your Azure encryption key when the cluster is destroyed. This value is only necessary if you intend to install the cluster with user-managed disk encryption.
+|String, for example `production_encryption_resource_group`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.subscriptionId`
+|Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
+|String, in the format `00000000-0000-0000-0000-000000000000`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskSizeGB`
+|The Azure disk size for the VM.
+|Integer that represents the size of the disk in GB. The default is `128`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskType`
+|Defines the type of disk.
+|`standard_LRS`, `premium_LRS`, or `standardSSD_LRS`. The default is `premium_LRS`.
+
+|`platform.azure.defaultMachinePlatform.type`
+|The Azure machine type for control plane and compute machines.
+|The Azure machine type.
+
+|`platform.azure.defaultMachinePlatform.vmNetworkingType`
+|The networking type for control plane and compute machines.
+|
+
+|`platform.azure.defaultMachinePlatform.zones`
+|For both compute machines and control plane machines, the list of availability zones that can be used. Examples: `cn-hangzhou-h`, `cn-hangzhou-j`
```

Comment:
How about rewording to `The availability zones where the installation program creates machines.`?

Reply:
I checked that `compute.platform.azure.zones` and `controlPlane.platform.azure.zones` are also missed, could you help to add them into the same table?

`compute.platform.azure.zones`
The availability zones where the installation program creates compute machines.

`controlPlane.platform.azure.zones` 
The availability zones where the installation program creates control plane machines.

Reply:
@jinyunma Updated the PR. can you PTAL? Thanks for the review :)